src/flags.zig

```diff
@@ -108,6 +108,8 @@ fn parse_commands(args: *std.process.ArgIterator, comptime Commands: type) Comma
 }
 
 fn parse_flags(args: *std.process.ArgIterator, comptime Flags: type) Flags {
+    @setEvalBranchQuota(5000);
```

Comment:
attonit, but `5_000`

Reply:
I'll fix that in an attopr to test this is working as expected!